src/main/java/staffconnect/logic/commands/UnmarkCommand.java

```diff
@@ -0,0 +1,112 @@
+import staffconnect.model.person.Module;
+import staffconnect.model.person.Name;
+import staffconnect.model.person.Person;
+import staffconnect.model.person.Phone;
+import staffconnect.model.person.Venue;
+import staffconnect.model.tag.Tag;
+
+/**
+ * Unmarks a person as favourite in the staff book.
+ */
+public class UnmarkCommand extends Command {
+
+    public static final String COMMAND_WORD = "unmark";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Unmarks a person as favourite in the staff book. "
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_UNMARK_PERSON_SUCCESS = "Person unmarked as favourite: %1$s";
+
+    public static final String MESSAGE_DUPLICATE_UNMARK = "This person has already been unmarked in the staff book.";
+
+    private final Index targetIndex;
+
+    /**
+     * @param targetIndex of the person in the filtered person list to unmark.
+     */
+    public UnmarkCommand(Index targetIndex) {
+        this.targetIndex = targetIndex;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        List<Person> lastShownList = model.getFilteredPersonList();
+
+        if (targetIndex.getZeroBased() >= lastShownList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+        }
+
+        Person personToUnmark = lastShownList.get(targetIndex.getZeroBased());
+
+        if (!personToUnmark.getFavourite().hasFavourite()) {
+            throw new CommandException(MESSAGE_DUPLICATE_UNMARK);
+        }
+
+        Person unmarkedPerson = createUnmarkedPerson(personToUnmark);
+
+        model.setPerson(personToUnmark, unmarkedPerson);
+        model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
+        return new CommandResult(String.format(MESSAGE_UNMARK_PERSON_SUCCESS, Messages.format(unmarkedPerson)));
+    }
+
+    /**
+     * Creates and returns a {@code Person} with the details of {@code personToUnmark}.
+     */
+    private static Person createUnmarkedPerson(Person personToUnmark) {
+        assert personToUnmark != null;
+
+        Name updatedName = personToUnmark.getName();
+        Phone updatedPhone = personToUnmark.getPhone();
+        Email updatedEmail = personToUnmark.getEmail();
+        Module updatedModule = personToUnmark.getModule();
+        Faculty updatedFaculty = personToUnmark.getFaculty();
+        Venue updatedVenue = personToUnmark.getVenue();
+        Set<Tag> updatedTags = personToUnmark.getTags();
+        Set<Availability> updatedAvailabilities = personToUnmark.getAvailabilities();
+        Favourite updatedFavourite = new Favourite(false);
+
+        return new Person(updatedName, updatedPhone, updatedEmail, updatedModule,
+            updatedFaculty, updatedVenue, updatedTags, updatedAvailabilities, updatedFavourite);
+    }
```

Comment:
I have a created a ```CopyPerson```` function for this in ```PersonUtil``` , essentially creating a copy of a person, do you want to modify your changes to that util code as well?

Reply:
Pardon me, but I don't seem to see the function that you've mentioned. Could you assist me in finding that first before I decide?

Reply:
They are part of #130

Reply:
I see what you meant. I'll put it in my backlog and modify it to use that after your pull request has been merged. Thank you for the suggestion. 👍